application/models/Tracker/Sites/ReadMangaToday.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1); defined('BASEPATH') OR exit('No direct script access allowed');
+
+class ReadMangaToday extends Base_Site_Model {
+	public $titleFormat   = '/^[a-zA-Z0-9_-]+$/';
+	public $chapterFormat = '/^[0-9\.]+$/';
+
+	public function getFullTitleURL(string $title_url) : string {
+		return "http://www.readmanga.today/{$title_url}";
+	}
+
+	public function getChapterData(string $title_url, string $chapter) : array {
+		//http://www.readmanga.today/saiki-kusuo-no-psi-nan/128
+		return [
+			'url'    => $this->getFullTitleURL($title_url).$chapter.'/',
+			'number' => "c{$chapter}"
+		];
+	}
+
+	public function getTitleData(string $title_url, bool $firstGet = FALSE) : ?array {
+		$titleData = [];
+
+		$fullURL = $this->getFullTitleURL($title_url);
+
+		$content = $this->get_content($fullURL);
+
+		$data = $this->parseTitleDataDOM(
+			$content,
+			$title_url,
+			"//body/div[@class, 'content']/div/div/div/div/div/div/div/div/h1",
```

Comment:
Does that mean that it's also incorrect in MangaCow's?
It has 
"//ul[contains(@class, 'mng_chp')]/li[1]/a[1]"
https://github.com/DakuTree/manga-tracker/blob/master/application/models/Tracker/Sites/MangaCow.php#L29